README.md

```diff
@@ -77,10 +67,15 @@ await _mailDispatcher.SendMail(
     to: new[] { new MailboxAddress("John Doe", "john.doe@example.tld") });
 ```
 
+# Convention based view loading
+Per default, the application tries to find all `*.Views.dll`'s found in the application base directory. It then loads all those dll's as application parts to the `IMvcCoreBuilder`. If you would like to opt-out or setup other `IMvcCoreBuilder` related things, you can use an overload:
+```csharp
+IServiceCollection AddMailDispatcher(this IServiceCollection serviceCollection, Action<MailDispatcherOptions> mailDispatcherOptionsBuilder, Action<IMvcCoreBuilder>? mvcCoreBuilderAction);
+```
+
```

Comment:
For consistencies sake it might be better to remove the convention based loading:

Overriding the default convention (loading all view assemblies) requires an additional argument being supplied to the extension method. This suggests that the properties supplied via the `IMvcCoreBuilder` are only available to the `MailDispatcher` instance, which is not the case, and might lead to confusion in other parts of the application.

Reply:
My bad as I implemented this default behaviour :p